python/core/additions/qgsgeometry.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+"""
+***************************************************************************
+    qgsgeometry.py
+    ---------------------
+    Date                 : May 2018
+    Copyright            : (C) 2018 by Denis Rouzaud
+    Email                : denis@opengis.ch
+***************************************************************************
+*                                                                         *
+*   This program is free software; you can redistribute it and/or modify  *
+*   it under the terms of the GNU General Public License as published by  *
+*   the Free Software Foundation; either version 2 of the License, or     *
+*   (at your option) any later version.                                   *
+*                                                                         *
+***************************************************************************
+"""
+
+
+def _geometryNonZero(self):
```

Comment:
is it an addition useful only for readability?

Reply:
see in `qgis.core.__init__.py`, this is used for `QgsGeometry.__bool__` so you can do `if myGeometry: ....`